### PR TITLE
feat: split docs file and import them in patternhub

### PR DESCRIPTION
**Pull request by nmerget**

### CODE REVIEW

File: showcases/patternhub/pages/components/index.mdx [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import DefaultPage from "../../components/default-page";
+import Readme from "../../../../packages/components/README.md";
```
Comment (mfranzke) [inline showcases/patternhub/pages/components/index.mdx:2]: we need to ensure that we don't make any external requests e.g. to shields.io, as we don't have enough information and no contract about / regarding their data security.

**Review by mfranzke — APPROVED** [review]

File: showcases/patternhub/pages/components/index.mdx (hunk 1 above)
Comment (nmerget) [reply showcases/patternhub/pages/components/index.mdx:2]: I guess this import will be compiled during the build process. Do you mean the content inside the components README.md?

Comment (mfranzke) [reply showcases/patternhub/pages/components/index.mdx:2]: doing a replacement by base64 or removal due to the build process would be perfectly fine from my point of view.

Comment (mfranzke) [reply showcases/patternhub/pages/components/index.mdx:2]: Created https://github.com/db-ui/mono/issues/723 for clarification.

**Pull request merged**